src/lava/magma/core/model/model.py

```diff
@@ -103,3 +105,9 @@ def __repr__(self):
             + " has tags   "
             + tags
         )
+
+    def on_var_update(self):
+        """This method is called if a Var is updated. It
+        can be used as callback function to calculate dependent
+        changes."""
+        pass
```

Comment:
Shouldn't this go to `AbstractPyProcessModel` rather ? As this is would only be feasible in Python ProcessModels

Reply:
Hm, probably not. Other backends might require different ways of updating the dependent parameters. See RegVars.